src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -169,17 +170,18 @@ internal abstract partial class DefinitionItem
             ImmutableArray<string> tags,
             ImmutableArray<TaggedText> displayParts,
             ImmutableArray<TaggedText> nameDisplayParts,
-            Solution solution, ISymbol symbol,
+            Project project,
```

Comment:
Is this place where we should be using [`SymbolAndProjectId`](http://source.roslyn.io/Microsoft.CodeAnalysis.Workspaces/R/d0997b8982d1805a.html).